lib/index.js

```diff
@@ -34,6 +34,8 @@ var CoverallsReporter = function(rootConfig, helper, logger) {
         filepath = path.resolve(rootConfig.basePath, reporter.dir);
         break;
       }
+    } else {
+     throw new Error("coverage reporter should precede coveralls");
```

Comment:
Ignore the 2nd comment, I think we just want to throw if `coverageReporter` isn't used